reads_filtration/module_wraper.py

```diff
@@ -10,8 +10,9 @@
 sys.path.insert(0, src_dir)
 
 from auxiliaries.pipeline_auxiliaries import fetch_cmd, wait_for_results
+from global_params import src_dir
```

Comment:
Is this needed? src_dir is defined in above lines and we always run the pipeline from main directory